src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -42,7 +42,7 @@ const AudioInputVFControl: React.FC<Props> = ({
   const meetingManager = useMeetingManager();
   const [isLoading, setIsLoading] = useState(false);
   // When the user click on Amazon Voice Focus option, the state will change.
-  const [isVoiceFocusOn, setIsVoiceFocusOn] = useState(false);
+  const [isVoiceFocusCheckboxOn, setIsVoiceFocusCheckboxOn] = useState(false);
```

Comment:
We do not use `Checkbox`, but, its a `Check` icon, so can we remove `Checkbox` and keep `isVoiceFocusCheckOn` ?

Reply:
I saw @simmkyu comment on the background blur on the same place:
>Two similar variable names-isVideoTransformOn and isVideoTransformEnabled-are confusing. You can make them more specific, e.g., isVideoTransformCheckBoxOn.

I also feel it is a little confusing between `isVoiceFocusOn` and `isVoiceFocusEnabled`, so rename it.

Reply:
Yep, this variable is internal. Do you mean to rename it as `isVoiceFocusChecked`?